pkgs/tools/misc/zsh-history-to-fish/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, fetchPypi
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "zsh-history-to-fish";
+  version = "0.3.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-expPuffZttyXNRreplPC5Ee/jfWAyOnmjTIMXONtrnw=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    click
+  ];
```

```suggestion
  ];
  
  # upstream has no tests
  doCheck = false;
  
  pythonImportsCheck = [
    "zsh_history_to_fish"
  ];
```
Would you please add `doCheck = false` and `pythonImportsCheck` since there are no tests?
pythonImportsCheck is useful for python application/package to simply check behavior.